rotkehlchen/tests/api/test_liquity.py

```diff
@@ -18,6 +19,8 @@
 LQTY_STAKING = string_to_evm_address('0x73C91af57C657DfD05a31DAcA7Bff1aEb5754629')
 LQTY_PROXY = string_to_evm_address('0x9476832d4687c14b2c1a04E2ee4693162a7340B6')
 ADDR_WITHOUT_TROVE = string_to_evm_address('0xA0446D8804611944F1B527eCD37d7dcbE442caba')
+JUSTIN = string_to_evm_address('0x3DdfA8eC3052539b6C9549F12cEA2C295cfF5296')
```

Comment:
Sun?

Reply:
the same, is one of the whales in liquity